_posts/2016-06-07-why-socrates-uk-is-not-just-a-conference-for-software-craftsmen.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+name: Why-SoCraTes-UK-is-not-just-a-conference-for-Software-Craftsmen
+title: 'Why SoCraTes UK is not just a conference for Software Craftsmen.'
+date: 2016-06-07 15:20:00 +00:00
+author: Giulia Mantuano
+image:
+    src: /assets/img/custom/blog/socrates.jpg
+tags:
+- socrates
+- conference
+- craftsmanship
+---
+
+Last Friday, I attended the international [Software Craftsmanship and Testing conference](http://socratesuk.org/index.html) at the enchanting Wotton House, in Dorking. I was the only Designer among about 100 Developers.
```

Comment:
Matthew reviewed it and I'd prefer to not touch it. Also because we would repeat the word "conference" twice in almost a line :)